mysql-test/r/venki_test.result

```diff
@@ -0,0 +1,12 @@
+CREATE TABLE t2(c2 INT PRIMARY KEY);
```

Comment:
I think we don't need this file :)

Reply:
Yeah. It must be a side effect of `git add mysql-test`. I will revert this change.